modules/internal/ChapelDistribution.chpl

```diff
@@ -780,7 +780,7 @@ module ChapelDistribution {
     proc dsiPostReallocate() {
     }
 
-    proc ~BaseArrOverRectangularDom() {
+    proc deinit() {
       // this is a bug workaround
```

Comment:
OOC, is the bug being worked around still present in the initializers world?

@mppf added it in a15c19cc128087, and @daviditen merged it in #6722, last January and July, respectively.

Reply:
I wondered that as well but did not have time to look into it.